packages/commonwealth/client/scripts/views/pages/Editor/codeBlockLanguages.ts

```diff
@@ -0,0 +1,107 @@
+export const codeBlockLanguages: { [key: string]: string } = {
```

Comment:
do we want to leave them commented? Imo, we should just leave those that we are gonna use

Reply:
Sort of fair but I had to pull these out of multiple places to figure out what languages it would support. This way we can just uncomment them if we want to use them.